go.mod

```diff
@@ -126,7 +128,7 @@ require (
 	github.com/prometheus/procfs v0.7.3 // indirect
 	github.com/rogpeppe/go-internal v1.6.1 // indirect
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
-	github.com/specularL2/specular/ops v0.0.0-20231129092153-13f66846bd07 // indirect
+	github.com/specularL2/specular/ops v0.0.0-20231216162439-3927ee049bbd // indirect
```

Comment:
can you try to remove this line to see if `go mod tidy` will add it back or not?

Reply:
it adds it back